### PR TITLE
Migrate remaining github tables

### LINKED ISSUE #2937: port the github service's tables to use normal DB tables
https://bugzilla.mozilla.org/show_bug.cgi?id=1630015

### PR DESCRIPTION
**Pull request by imbstack**
Github Bug/Issue: Fixes #2937

If this has a bunch of red tests in the morning then feel free to skip reviewing but if it's green then go ahead and take a peek!

### CODE REVIEW

File: db/test/helper.js [hunk 1]
```diff
@@ -325,42 +325,46 @@ exports.testEntityTable = ({
         });
       }
 
-      for (let {condition, expectedSamples} of scanConditions) {
-        test(`scan ${JSON.stringify(condition)} (${expectedSamples.length} results)`, async function() {
-          const res = await Entity.scan(condition);
-          const expected = expectedSamples.map(n => samples[n]);
-          assert.deepEqual(res.entries.map(e => e._properties), expected);
-        });
+			if (!notImplemented.includes('scanning')) {
```
Comment (helfi92) [inline db/test/helper.js:328]: Fix indentation.

File: db/versions/0025-migration.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  create table github_checks
+  as
+    select
+      (value ->> 'taskGroupId')::text as task_group_id,
+      (value ->> 'taskId')::text as task_id,
+      (value ->> 'checkSuiteId')::text as check_suite_id,
+      (value ->> 'checkRunId')::text as check_run_id
+    from taskcluster_check_runs_entities;
+  alter table github_checks add primary key (task_group_id, task_id);
```
[the remark this answers is not in the document]
Comment (imbstack) [reply db/versions/0025-migration.sql:18]: ah yeah, totally spaced on that

File: services/github/src/api.js [hunk 3]
```diff
@@ -269,10 +269,10 @@ builder.declare({
 
       case 'integration_installation':
       // Creates a new entity or overwrites an existing one
-        await this.OwnersDirectory.create({
-          installationId: body.installation.id,
-          owner: body.installation.account.login,
-        }, true);
+        await this.db.fns.create_github_integration(
```
[the remark this answers is not in the document]
Comment (imbstack) [reply services/github/src/api.js:272]: huh, I guess that isn't tested

**Comment by imbstack** [comment]
assuming that this makes sense, let me know what number migration I should rebase to!

**Comment by djmitche** [comment]
I think we're close to shipping a point-in-time release with #3242.  Let's get that out the door, then land this, thereby only rebasing it once :)

**Comment by djmitche** [comment]
versions 0036 and 0037 are _all yours_

**Comment by djmitche** [comment]
Rebased and the changelog updated with timing for the migration (about 10s).  I'll merge once it's green.

**Pull request merged**